src/elfpy/user.py

```diff
@@ -133,44 +186,129 @@ def get_trade(self):
         return action_list
 
     def update_spend(self):
```

Comment:
add function comments, remove commented prints

Reply:
keeping commented prints, so we can add them back in w/ attached logging TODO: